learn-responsive-web-design-by-building-a-piano/index03.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<body>
+
+ <div>
+     <div></div>
+ </div>
+
+</body>
+</html>
+
+<!-- 
+    To distiguish between both divs and to help us
```

Comment:
Students should be familiar with adding `id` and `class` attributes to HTML elements by this point in the curriculum. You can break this into two lessons, the first with instructions like:

Give the outer `<div>` an `id` attribute with the value `piano`. Here is an example: 
```
<div id="guitar">
  <div></div>
</div>
```

The second lesson could instruct the student to give the inner `<div>` element a `class` with the value `keys`.